app/helpers/quadicon_helper.rb

```diff
@@ -91,6 +79,95 @@ def img_for_host_vendor(item)
 
   private
 
+  def render_quadicon_text(item, row)
+    return unless item
+    db = db_from_item(item)
+
+    if @embedded && !@showlinks
+      column = case db
+                 when "MiqCimInstance"   then 'evm_display_name'
+                 when "ConfiguredSystem" then 'hostname'
+                 else 'name'
+                 end
+      content_tag(:span, :title => h(row[column])) do
+        truncate_for_quad(h(row[column]))
+      end
+    else
+      if !@listicon.nil? && (@vm || @host || @storage)
+        # if sub-item is being shown
+        if @listicon == "scan_history"
+          href_link = url_for_item_quad_text(@vm, @id, @listicon.pluralize)
+          content_tag(:a, truncate_for_quad(row['started_on'].to_s),
+                      :href => href_link, :title => h(row['started_on']))
```

Comment:
(I know this is just a step along the way, but) the alternation between `link_to` and `content_tag` seems jarring